aspnetcore/mvc/models/validation.md

```diff
@@ -419,6 +419,8 @@ Built-in validation attributes include:
 * `[Url]`: Validates that the property has a URL format.
 * `[Remote]`: Validates input on the client by calling an action method on the server. See [`[Remote]` attribute](#remote-attribute) for details about this attribute's behavior.
 
+The `RegularExpression` attribute uses [ECMAScript](/dotnet/standard/base-types/regular-expression-options#ecmascript-matching-behavior) options for matching on client side validation. For more informaion, see [this GitHub issue](https://github.com/dotnet/corefx/issues/42487).
```

Comment:
A couple notes:

1. Might be worth making this a [`[!NOTE]` callout](https://dotnet.github.io/docfx/spec/docfx_flavored_markdown.html#note-warningtipimportant)
2. I'd rephrase this a bit. It implies the "RegularExpression attribute" is doing something, when it's really just a piece of data. Better to rephrase around something like "When using `[RegularExpression]` validation in an ASP.NET or ASP.NET Core application ...".
3. Seems appropriate to make clear *when and why* this happens (when: doing client-side validation, why: because the validation rules are translated to JavaScript)

Reply:
@anurse we avoid chrome/bling/note when possible.

Reply:
Sure, then not a `[!NOTE]` but the rest still applies.

Reply:
@anurse  I applied the rest. Do you want to review again or can I S&M.